src/react/__tests__/base/rfc1.test.tsx

```diff
@@ -61,9 +61,10 @@ test('rfc1 example implementation', async() => {
   const CurrentText = ({prefix}) => {
     const {text} = useStore(store)
     return (
-      <p>
-        {prefix} {text}
-      </p>
+      <div>
+        <p>{prefix}</p>
+        <p>{text}</p>
+      </div>
```

Comment:
See comment for https://github.com/effector/effector/pull/655/commits/004da3e771d69b7898c6b02626f24dca9c621e2d